knack-load/knackload.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import os
+import io
+import json
+
+# all our little secrets ...
+__APIKEY__ = None
+__APIID__ = None
+
+# where the knack data lives
+__APIURI__ = None
+
+#the headers, since they never change from request to request, since we are only
+# doing POSTS here
+__HEADERS__ = None
+
+# set the credentials
+def setCreds():
+    # XXX TODO: verfiy this works on whatever app platform this is going to run on
+    global __APIKEY__
```

Comment:
generally prefer that we not use globals. whatever web framework we use (Django or Flask, probably), will have a way for us to load these values up in settings.